efficientdet/det_model_fn.py

```diff
@@ -289,12 +286,14 @@ class and box losses from all levels.
       cls_targets_at_level = tf.reshape(cls_targets_at_level,
                                         [bs, width, height, -1])
     box_targets_at_level = labels['box_targets_%d' % level]
-    cls_loss = _classification_loss(
-        cls_outputs[level],
-        cls_targets_at_level,
-        num_positives_sum,
-        alpha=params['alpha'],
-        gamma=params['gamma'])
+
+    cls_loss = focal_loss(
```

Comment:
focal_loss has some hard-coded numbers (like "positive_label_mask = tf.equal(targets, 1.0)"), you may want to update that?

You can also borrow some ideas or reuse the tfa.focal_loss? 

@fsx950223 seems to be familiar to tfa (tf addons)

Reply:
Sure, I can update the focal loss function to be more like tfa focal_loss.

I would definitely appreciate @fsx950223 's insights.

Reply:
I updated the focal loss function by adapting logic from the tfa version. Do you like this new version better? If so, can you please check the algebra? If I made a mistake in the math it would be pretty hard to catch at runtime. Thanks

Reply:
For reference: https://github.com/tensorflow/addons/blob/v0.10.0/tensorflow_addons/losses/focal_loss.py#L90